### ISSUE TITLE
Mass Protest Cross Countries by Paula Hierzi

### ISSUE THREAD

**Issue by rjeantet** [TechLabs-Berlin/project_proposals#13]
### What is your project's name?
Mass Protest Cross Countries
### What's the problem you are trying to solve?
Analyzing when is a Protest swaping over to Cross Borders? When does it stay Regional ?
### What's your idea for a solution to the given "problem"?
Looking at Protests that went " Viral" like "Un Violador in Tu Camino" that started in a small town in Valparaíso and went until the Norwegian Parliamant and further.
### Which tracks do you think could be involved?
DL/DS, WD, UX
### *How do you imagine Deep Learning and/or Data Science could contribute to this project?*
Data Science can contribute in Alayzing the Data:
https://dataverse.harvard.edu/dataverse/MMdata
https://poldem.eui.eu/data-overview/
And searching for patterns or timing of Protest that spread the world.
### *How do you imagine User Experience Design could contribute to this project?*
UX has the role of making this Data clear and easily readable and presentable. That people are excited to be on our Webpage
### *How do you imagine Web Development could contribute to this project?*
WD would be responsible to make our Website. To see all the Data we have Analyzed to connect links to the resources we used .....
### What are the most important challenges & bottlenecks of your project?
To find the connections cross countries.
To be able to present our found Analyzations
### What are the most important success factors of your project?
Finding a way to present the found data in an appealing easy Form. Connecting the Dots of the Data. Finding the right Data to make this visible( Maybe my links are enough ?)
### Share your resources here:
https://dataverse.harvard.edu/dataverse/MMdata
https://poldem.eui.eu/data-overview/
### Do you have any other remarks?
I have been thinking of Analyzing different Data:
1.Mass Protest: I have found most Data Sets ...
I have other DatSets in mind I could imagine Analyzing as well.
2.Single Parenting> which Jobs are more accessible for sole Caretakers?  What is affecting this accesibility?
This website from a friend is an inspiration:
https://re-dance.work/team/
But I have not really found accessible Data especially if you look at Intersectional Perspectives as Race, Class, Gender, Sexuality.
3. Analyzing Data from the Gaza Strip.
https://data.humdata.org/m/event/opt-israel-hostilities?page=2#datasets-section
How did this war happen?  Who is affected from it?
....
Do you have any other Socio- political for DataSets Ideas you can comment below!

**Comment by rjeantet** [comment]
Hi @Rustingheart , thanks for submitting this new idea 👍🏼 

As this project idea relies strongly on data exploration and visualization, team constitution and project roadmap will have some specificities. Here are a few pointers:

### 1. Discovery & Project definition:
A significant part of UX role is to help define the audience (who needs this problem solved), then validate the assumptions (how the idea will help solve the problem), before designing the solution and finally testing it.

It will probably help fellow techies to project themselves into the idea if you could add some details regarding those points.
One way you can solve this is by trying the exercise of summing up the idea into a one-sentence pitch, like this:
"The project Mass Protest Cross Countries is developing [a defined offering] to help [a defined audience] [solve a problem].”

Going through these steps will greatly help define the goals, scope, and features of the project.

- [x] try this exercise having in mind your audience (researchers? students? politically engaged people...etc...?) and the **why** they need this solution (prevent protests from crossing borders? Or on the contrary encourage protests to go viral...etc...)

### 2. App features and track requirements:
_At this stage_, because the project idea focuses mostly on displaying data, there would not be necessarily a need for user and authentication management. Also since there is no potentially sensitive data to handle, the API requests could be handled directly client side talking with the DS python server. This being said,

- Try thinking about and listing the potential for feature expansion, such as adding user-generated content or social sharing capabilities, as this will impact the need for WD backend or not.
- The Track requirements for DS involve the implementation of some machine learning algorithm. Here to Try to think of use case and features that would involve them (predictive analysis...etc...)
- [x] Create a list of "(3-4) must-have" and additional "nice-to-have" features.

### 3. Design & Collaboration
Creating the graphs and visualization is a crucial part of this project.
A typical design process starts with UX designing high-fidelity wireframes and providing them to devs to implement UI and connect API's.
In this case, whereas UX still holds full freedom regarding the global app design, the approach regarding the data visualization will probably be slightly different:
1. After exploring data, DS tells UX what is available to use
2. UX (in collaboration with DS), defines the type of charts needed
3. WD frontend investigates those charts and libraries to be implemented (for example [Recharts](https://recharts.org/), [nivo](https://github.com/plouc/nivo)...etc...)
4. Within the design constraints of the library chosen, UX defines styling
5. WD frontend implements the UI
6. In parallel DS setup the python server and WD frontend & DS work together on connecting the API for data retrieval

- [x] No action is needed here, simply recommendations on how to organize the work within team members on the specific subject of data design.

### 4. Project scope
This goes along with the project definition. You are mentioning possible different datasets. For the project to be clearly defined, ideally you would try focusing the thought process on the problem you want to solve first, and then only the subject of the dataset.

I reckon that was quite a lot of insights!! 
I hope it helps! Do not hesitate if you have additional questions :)

**Comment by Rustingheart** [comment]
### 1. Discovery & Project definition: 

The project "Mass Protest Cross Countries" wants to find tools and solutions to make a Protest go viral!!
With examples from past events, we might be able to find out what makes a topic so accessible, and pressing that it can go viral. 
Maybe we can sort out which conditions helped reach more audiences and convince people.
Every Country's conditions and culture for Protest are different problems. But some movements even target the whole world such as: the Decolonisation Movement, Feminist Movement, BlackLivesMatter, Abolitionist Movement, FridaysForFuture, ... . 
Is there a way of making them more powerful? 
Is there a way of predicting what can happen if there is a certain amount of people mobilized? 
TikTok, Instagram, Facebook and other social networks make it possible to spread information easily. And play a big role in a Protest going Viral.
Maybe we can analyze some Twitter-Data about a certain time span and certain words being used like 'UnVioladorInTuCamino'? 
>Presenting the found Data in an accessible way. 

This project also could have a Berlin specific side-page to collect information from regional Protests to specify topics taking place.
* Sometimes it can be confusing when and where protests take space. 
* You want to know if there is a Protest for a certain topic is happening in your local area.
* maybe you want to spread your information of a protest you are organizing. 

> Make a questionnaire where people can give in information about a protest happening.
> Collect this Data and sorting it after topics
> Make sure there is a link to a real event happening so it can not be spammed. 
> Make our idea clear for the User and simply structured so the User can access/give the information needed. 

### Our USERS:
People who want to learn more about a certain Protest how and where they happened  
- (What were the conditions in different Countries?)(Nice to have?)
People who want to give and have information about Protests happening in Berlin. 

### 2. App features and track requirements: 

**UX:** 
Landing Page for the whole Project for everyone to understand what we offer:
BERLIN
- Create a Form to "Input Data"
> Info about Protest: Flyer, Link, Time, Location, Topic(s), Info, ...
- Design a Page INFO: 
where the "Input Data" is presented in a clear way 
> With sorting possibilities
RESEARCH
- Design a Landing Page to make our researched DataSets visible and accesible.
- Design a Small Page for Resources and links 

**WD**
FrontEnd:
- Make a Main Landing Page for "Mass Protests"
- 2x Side Landing Pages: > 1. Berlin; 2. Research
- 2x Seperate Pages for Berlin: 1. Input; 2. Info
BackEnd:
Berlin: 
Connecting Input to Info? 

**DS**
BERLIN:
- Collecting Data Formatting, Sorting
RESEARCH:
- Getting Data
- Analyzing Data
- Presenting it

**TEAM**: I am always very happy about ideas you want to add!!!

**Comment by rjeantet** [comment]
Wow @Rustingheart !

Thank you for being so reactive, and applying thoroughly the feedback!

If I were to sum up, the project "Mass Protest Cross Countries" provides tools and solutions to make a Protest go viral, by:
- analyzing patterns of big historical protests and predicting potential future viral protests (ML algorithm)
- inform a broader audience about those protests by presenting them under nice, interactive visualization
- providing local citizens with the possibility of submitting and following the evolution of local protests

Do you agree with that? 
Of course this is the rough idea and as you pointed, your teammates and you will brainstorm further and ideate as a team.

NB: Because the project offers the possibility to create protests information within the app, a backend developer is definitely needed for security, user authentication and management.

The team composition suggestion is therefore:

- [ ]  1 to 2 UX designers
- [ ]  1 WD frontEnd
- [ ] 1 WD backEnd
- [ ] 2 DS

Moving your project to accepted! 💯

**Comment by irembyrm** [comment]
Hey Paula, It is really interesting project :) I would be very happy to contribute as DS member :)

**Comment by Rustingheart** [comment]
So cool thank you for reviewing @rjeantet!
I am so happy you are interest @irembyrm + @annir3 !!
DL if you have any idea how to join add a comment !
I heard that there is accesible Twitter Data maybe DL track you can filter out RESEARCH related posts ?

https://imerit.net/blog/top-25-twitter-datasets-for-natural-language-processing-and-machine-learning-all-pbm/

**Comment by annir3** [comment]
Hi Paula :) I'm in the WD (FrontEnd) track and would love to join.

**Comment by Rustingheart** [comment]
https://www.dezim-institut.de/aktuelles/aktuelles-detail/protestdataeu-ein-online-tool-um-protestdynamiken-zu-begreifen/

https://de.statista.com/statistik/daten/studie/1367015/umfrage/gruppen-von-protestbewegungen-weltweit/

https://www.protestdata.eu/methods

Those are a  much bigger scope but interesting to see

Also some info how Data is collected

https://www.annualreviews.org/doi/abs/10.1146/annurev.soc.30.012703.110603

**Comment by werka93** [comment]
Allo Paula, I will be interested to be 1 of 2 UX designer

**Comment by Rustingheart** [comment]
Hello @werka93 are you gonna be in our group later tonight? I could not find you on slack. Did you state your preference for this group in the form?

**Comment by werka93** [comment]
Allo Rustingheart, sure, I will be there. On Slack I‘m Weronika Ajimati. I like this project idea.
